src/android/index.jsx

```diff
@@ -247,44 +247,22 @@ class Android extends Component {
             <Grid item xs={6}>
               <PerfherderGraphContainer
                 timeDomain={timeDomain}
-                title="Speedometer"
-                series={[
-                  {
-                    label: 'Moto G5 (arm7)',
+                title="Speedometer Moto G5"
+                series={selectFrom(COMBOS)
+                  .where({ suite: 'speedometer' })
+                  .map(({ browserLabel, filter: browserSuiteFilter }) => ({
+                    label: browserLabel,
                     filter: {
                       and: [
                         { missing: 'test' },
                         {
                           prefix: { platform: 'android-hw-g5-7-0-arm7-api-16' },
                         },
```

Comment:
Please add a `PLATFORMS` config variable, and use it to generate one chart for each suite:

```
PLATFORMS.map(({platformLabel: label, platformFilter: filter})=>(
              <PerfherderGraphContainer
                timeDomain={timeDomain}
                title={"Speedometer"+ platformLabel}
                series={selectFrom(COMBOS)
                  .where({ suite: 'speedometer' })
                  .map(({ browserLabel, filter: browserSuiteFilter }) => ({
                    label: browserLabel,
                    filter: {
                      and: [
                        { missing: 'test' },
                        platformFilter,
                        browserSuiteFilter
                      ]
                    }
     ....
))

Reply:
Yes, I have pushed the code before I move into `PLATFORMS` config variable because the `diff` would be tough to keep track.  Now, I will start working on this variable.

Reply:
@klahnakoski I generated the chart using `PLATFORMS` variable. `Unity WebGl Pixel 2` has no chart to render.